WordPressKit/WordPressComOAuthClient.swift

```diff
@@ -272,7 +272,7 @@ public final class WordPressComOAuthClient: NSObject {
             .responseJSON(completionHandler: { response in
                 switch response.result {
                 case .success(let responseObject):
-                    DDLogVerbose("Received Social Login Oauth response: \(self.cleanedUpResponseForLogging(responseObject as AnyObject? ?? "nil" as AnyObject))")
```

Comment:
Is this removal related to the migration or something else?

I tried using the original line and it seems to work fine in Swift 5.

Reply:
Good question - this removal is the bugfix for #149. Social logins should not expose the bearer token.

Reply:
I made this addition because it seemed important to fix.